src/plugins/cssSnippetManager.tsx

```diff
@@ -0,0 +1,374 @@
+        forceUpdate();
+
+        Toasts.show({
+            message: `Successfully ${isUpdate ? "updated" : "applied"} the snippet!`,
+            type: Toasts.Type.SUCCESS,
+            id: Toasts.genId()
+        });
+    };
+
+    const removeSnippet = () => {
+        snippet && (Vencord.Plugins.plugins.CssSnippetManager as any)?.removeStyle(snippet);
+
+        Settings.plugins.CssSnippetManager.cssSnippets = getSnippets().filter(snip => snip.messageId !== message.id);
+        forceUpdate();
+
+        Toasts.show({
+            message: "Successfully deleted the snippet!",
+            type: Toasts.Type.SUCCESS,
+            id: Toasts.genId()
+        });
+    };
+
+    return (
+        <div>{
+            snippet !== undefined ? (
+                <div style={{ display: "flex", flexDirection: "row", columnGap: "4%" }}>
+                    {timestamp !== snippet.editedTimestamp &&
+                        <Button
+                            className={getCssClassNames().colorGreen}
+                            onClick={applyOrUpdate}
+                            size={Button.Sizes.SMALL}
+                            style={{ paddingInline: "0" }} // Discord bad, the text does not fit due to the padding ;-;
+                        >
+                            Update
+                        </Button>}
+                    <Button
+                        className={getCssClassNames().colorRed}
+                        onClick={removeSnippet}
+                        size={Button.Sizes.SMALL}
+                        style={{ paddingInline: "0" }} // Discord bad, the text does not fit due to the padding ;-;
+                    >
+                        Remove
+                    </Button>
+                </div>
+            ) : (
+                <Button
+                    onClick={applyOrUpdate}
+                    size={Button.Sizes.SMALL}
+                >
+                    Apply
+                </Button>
+            )
+        }</div>
+    );
+};
+
+
+export default definePlugin({
+    name: "CssSnippetManager",
+    description: "Allows the user to apply CSS snippets from messages.",
+    authors: [Devs.Arjix],
+    dependencies: ["MessageAccessoriesAPI"],
+    options: {
+        cssSnippets: {
+            type: OptionType.COMPONENT,
+            description: "",
+            component: SnippetMgrSettings,
+            onChange() {
+                const plugin = Vencord.Plugins.plugins.CssSnippetManager!!;
+                plugin.stop!!();
+                plugin.start!!();
+            },
+        },
+        channels: {
+            type: OptionType.STRING,
+            default: "1028106818368589824",
+            description: "The channel IDs you want to install snippets from. Seperated using a comma. (e.g. 123,456)",
+            isValid: value => value.split(",").map(id => /\d+/.test(id.trim()) || id.trim().length === 0).every(id => id),
```

Comment:
perhaps allow an optional space between elements?
```suggestion
            isValid: value => value.split(/, ?/).map(id => /\d+/.test(id.trim()) || id.trim().length === 0).every(id => id),
```

Reply:
I use .trim() tho...